core/object/class_db.cpp

```diff
@@ -514,6 +515,17 @@ Object *ClassDB::_instantiate_internal(const StringName &p_class, bool p_require
 #endif
 		return obj;
 	} else {
+#ifdef TOOLS_ENABLED
+		if (!p_require_real_class && ti->is_runtime && Engine::get_singleton()->is_editor_hint()) {
+			if (!ti->inherits_ptr || !ti->inherits_ptr->creation_func) {
+				ERR_PRINT_ONCE(vformat("Cannot make a placeholder instance of runtime class %s because it's parent cannot be constructed.", ti->name));
```

Comment:
```suggestion
				ERR_PRINT_ONCE(vformat("Cannot make a placeholder instance of runtime class %s because its parent cannot be constructed.", ti->name));
```

Reply:
Thanks! Fixed in my latest push